templates/travel/partials/footer/footer.snip.html

```diff
@@ -0,0 +1,108 @@
+{{!
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+<!-- Footer -->
+<footer class="travel-footer overflow-hidden">
+  <section class="travel-footer-blog max-width-3 mx-auto px1 md-hide lg-hide">
+    <div class="flex md-mx2 my3">
+      <amp-img
+        class="rounded relative flex-none mx1"
+        width="100"
+        height="100"
+        src="../../../img/travel/blogpost-thumbnail.jpg"
+      ></amp-img>
+      <div class="flex-auto">
+        <div class="h3 bold line-height-2 mb1">From the blog</div>
+        <p class="travel-footer-blog-text my1">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit amet dolor set. <a class="travel-link" href="#">Read more</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <div class="relative bg-black">
+
+    <!-- Angle -->
+    <div class="travel-footer-angle-block absolute xs-hide sm-hide"></div>
+    <div class="travel-footer-angle xs-hide sm-hide"></div>
+    <!--/ Angle -->
+
+    <!-- Right column -->
+    <div class="travel-newsletter-signup">
+      <header class="max-width-3 mx-auto px1 md-px2">
+        <h4 class="travel-footer-right-column-heading travel-spacing-none h2 mb3 blue">Want travel deals<br>straight to your inbox?</h4>
+        <div class="h4 bold mb1">Sign up to our newsletter</div>
+      </header>
+      <div class="relative">
```

Comment:
"i think this div can be combined with its child. there so many wrapping divs all over :(" A: This div holds a child that needs to be absolutely positioned with respect to its sibling